test/common/upstream/health_checker_impl_test.cc

```diff
@@ -2187,9 +2234,9 @@ class TcpHealthCheckerImplTest : public testing::Test {
     }
     )EOF";
 
-    health_checker_.reset(new TcpHealthCheckerImpl(*cluster_, parseHealthCheckFromV1Json(json),
-                                                   dispatcher_, runtime_, random_,
-                                                   HealthCheckEventLoggerPtr(event_logger_)));
+    health_checker_.reset(
+        new TcpHealthCheckerImpl(*cluster_, parseHealthCheckFromV1Json(json.str()), dispatcher_,
```

Comment:
As long as you are in here would you mind updating these tests to use v2 YAML instead of JSON as @derekargueta has been doing elsewhere? This would really help our v1 deprecation efforts.

Reply:
Done, these were the last users of parseHealthCheckFromV1Json so I removed that too